src/Raven.Client/Documents/Operations/ConnectionStrings/GetConnectionStringsOperation.cs

```diff
@@ -86,51 +87,32 @@ public sealed class GetConnectionStringsResult
         public Dictionary<string, SnowflakeConnectionString> SnowflakeConnectionStrings { get; set; }
-
+        public Dictionary<string, AiConnectionString> AiConnectionStrings { get; set; }
+        
         public DynamicJsonValue ToJson()
         {
-            var ravenConnections = new DynamicJsonValue();
-            var sqlConnections = new DynamicJsonValue();
-            var elasticSearchConnections = new DynamicJsonValue();
-            var olapConnections = new DynamicJsonValue();
-            var queueConnections = new DynamicJsonValue();
-            var snowflakeConnections = new DynamicJsonValue();
-
-            foreach (var kvp in RavenConnectionStrings)
-            {
-                ravenConnections[kvp.Key] = kvp.Value.ToJson();
-            }
-            foreach (var kvp in SqlConnectionStrings)
-            {
-                sqlConnections[kvp.Key] = kvp.Value.ToJson();
-            }
-            foreach (var kvp in ElasticSearchConnectionStrings)
-            {
-                elasticSearchConnections[kvp.Key] = kvp.Value.ToJson();
-            }
-            foreach (var kvp in OlapConnectionStrings)
-            {
-                olapConnections[kvp.Key] = kvp.Value.ToJson();
-            }
-            foreach (var kvp in QueueConnectionStrings)
-            {
-                queueConnections[kvp.Key] = kvp.Value.ToJson();
-            }
-            foreach (var kvp in SnowflakeConnectionStrings)
-            {
-                snowflakeConnections[kvp.Key] = kvp.Value.ToJson();
-            }
+            var result = new DynamicJsonValue();
+
+            AddConnections(RavenConnectionStrings);
+            AddConnections(SqlConnectionStrings);
+            AddConnections(OlapConnectionStrings);
+            AddConnections(ElasticSearchConnectionStrings);
+            AddConnections(QueueConnectionStrings);
+            AddConnections(SnowflakeConnectionStrings);
+            AddConnections(AiConnectionStrings);
 
+            return result;
 
-            return new DynamicJsonValue
+            void AddConnections<T>(Dictionary<string, T> connectionStrings) where T : IDynamicJsonValueConvertible
             {
-                [nameof(RavenConnectionStrings)] = ravenConnections,
-                [nameof(SqlConnectionStrings)] = sqlConnections,
-                [nameof(OlapConnectionStrings)] = olapConnections,
-                [nameof(ElasticSearchConnectionStrings)] = elasticSearchConnections,
-                [nameof(QueueConnectionStrings)] = queueConnections,
-                [nameof(SnowflakeConnectionStrings)] = snowflakeConnections,
-            };
+                var jsonConnections = new DynamicJsonValue();
+                
+                foreach (var kvp in connectionStrings)
+                    jsonConnections[kvp.Key] = kvp.Value.ToJson();
+                
+                var name = typeof(T).Name;
+                result[name] = jsonConnections;
```

Comment:
this is not correct, should be a separate parameter based on `nameof()` like in the original code